website/docs/r/redis_linked_server.html.markdown

```diff
@@ -0,0 +1,104 @@
+---
+subcategory: "Redis"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_redis_linked_server"
+description: |-
+  Manages a Redis Linked Server.
+---
+
+# azurerm_redis_linked_server
+
+Manages a Redis Linked Server (ie Geo Location)
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "pri" {
+  name     = "example-resourcesi-pri"
+  location = "East US"
+}
+
+resource "azurerm_redis_cache" "pri" {
+  name                = "example-cache1"
+  location            = azurerm_resource_group.pri.location
+  resource_group_name = azurerm_resource_group.pri.name
+  capacity            = 1
+  family              = "P"
+  sku_name            = "Premium"
+  enable_non_ssl_port = false
+
+  redis_configuration {
+    maxmemory_reserved = 2
+    maxmemory_delta    = 2
+    maxmemory_policy   = "allkeys-lru"
+  }
+}
+
+resource "azurerm_resource_group" "sec" {
+  name     = "example-resources-sec"
+  location = "West US"
+}
+
+resource "azurerm_redis_cache" "sec" {
+  name                = "example-cache2"
+  location            = azurerm_resource_group.sec.location
+  resource_group_name = azurerm_resource_group.sec.name
+  capacity            = 1
+  family              = "P"
+  sku_name            = "Premium"
+  enable_non_ssl_port = false
+
+  redis_configuration {
+    maxmemory_reserved = 2
+    maxmemory_delta    = 2
+    maxmemory_policy   = "allkeys-lru"
+  }
+}
+
+resource "azurerm_redis_linked_server" "link" {
+  redis_cache_name            = azurerm_redis_cache.pri.name
+  resource_group_name         = azurerm_redis_cache.pri.resource_group_name
+  linked_redis_cache_id       = azurerm_redis_cache.sec.id
+  linked_redis_cache_location = azurerm_redis_cache.sec.location
+  server_role                 = "Secondary"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `linked_redis_cache_id` - (Required) The ID of the linked Redis ache. Changing this forces a new Redis to be created.
+
+* `linked_redis_cache_location` - (Required) The location of the linked Redis cache. Changing this forces a new Redis to be created.
+
+* `redis_cache_name` - (Required) The name of Redis cache. Changing this forces a new Redis to be created.
```

Comment:
would this make more sense as
```suggestion
* `target_redis_cache_name` - (Required) The name of Redis cache to link with. Changing this forces a new Redis to be created.
```

Reply:
Yes we can as usual the clarity of the naming coming from upstream isn't great.  Not that i am any better. Typically why I stick closest to the API for starters.  I appreciate the feedback.  I was hoping the example would be clear by selecting highlighting primary and secondary.